Common/src/gRPC/ListResultsRequestExt.cs

```diff
@@ -39,6 +39,7 @@ public static class ListResultsRequestExt
     => sort.Field.FieldCase switch
        {
          ResultField.FieldOneofCase.ResultRawField => sort.Field.ResultRawField.Field.ToField(),
+         ResultField.FieldOneofCase.None           => throw new ArgumentOutOfRangeException(nameof(sort)),
```

Comment:
That's already handled by the "_" case isn't it ?

Reply:
yes but resharper wants to see ervery arm in the switch case